kafka_consumer_freshness_tracker/src/test/java/com/tesla/data/consumer/freshness/ConsumerFreshnessTest.java

```diff
@@ -272,6 +277,124 @@ public void testBurrowMissingConsumerGroupPartitionEndOffsetMarksMissing() throw
     });
   }
 
+  @Test
```

Comment:
Can you also add a test that covers the metric update?

Reply:
added